package.json

```diff
@@ -43,8 +43,6 @@
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
     "mozilla-lint": "addons-linter dist/firefox",
-    "ui": "npm run test:flat:build:states && beefy development/ui-dev.js:bundle.js --live --open --index=./development/index.html --cwd ./",
-    "mock": "beefy development/mock-dev.js:bundle.js --live --open --index=./development/index.html --cwd ./",
```

Comment:
Since you're getting rid of this two commands,  What should we do with:

- https://github.com/MetaMask/metamask-extension/blob/develop/docs/ui-mock-mode.md
- https://github.com/MetaMask/metamask-extension/blob/develop/docs/ui-dev-mode.md
- https://github.com/MetaMask/metamask-extension/blob/develop/development/mock-dev.js
- https://github.com/MetaMask/metamask-extension/blob/develop/development/ui-dev.js

Reply:
Yup, I'll delete those in an upcoming PR with the rest of the old UI setup 👍